### PR TITLE
Notes from test install

### PR DESCRIPTION
**Pull request by abought**
Add a file describing local installation and how to create a persistent development, separate from ephemeral test directories.

### CODE REVIEW

**Comment by pjvandehaar** [comment]
Adding this line to `./config.py` will avoid downloading dbSNP:

```
cache = '/Users/peter/pheweb/tests/input_files/fake-cache'
```

Is this worth mentioning?

**Comment by abought** [comment]
> Is this worth mentioning?

Overall, yes! Would the fake cache change how the generated plots look / behave? I imagine the "development environment" scenario would be most useful for someone who needed to tweak the UI, with a quick way to get a decent plot for testing purposes. 

Anything that could save an hour (!) up front is super useful- we should mention it, even if not as a default recommendation. Maybe a footnote?

I would leave off the absolute path in the example, though- not everyone is `Peter`. For the most part these instructions call for commands to be run from the cloned repository root, and the goal is for anyone who opens this repo to use the examples and artifacts equally. (see [sample file that gave me pause](https://github.com/statgen/pheweb/blob/d704acbe32e80ba0087b7da38f6d11afd4531eac/tests/input_files/phenolist/phenolist-example1.csv#L3))

File: etc/detailed-development-instructions.md [hunk 1]
```diff
@@ -25,7 +25,14 @@ The data used for unit tests can also provide the basis for a persistent local d
 
 First, generate a working *pheno-list.json* file:
 
-`pheweb phenolist import-phenolist tests/input_files/categories.csv`
+```
```
Comment (abought) [inline etc/detailed-development-instructions.md]: Thanks for catching this- I misread the option in the README. :)

File: pheweb/load/top_hits.py [hunk 2]
```diff
@@ -53,7 +53,7 @@ def run(argv):
 just the top phenotype for each, use `pheweb top-loci`.
 '''.format(out_filepath_json,
            out_filepath_tsv,
-           '{:0.0e}'.format(conf.top_hits_pval_cutoff).replace('e-0', 'e-'),
+           '{:0.0e}'.format(min(conf.top_hits_pval_cutoff, conf.manhattan_peak_pval_threshold)).replace('e-0', 'e-'),
```
Comment (abought) [inline pheweb/load/top_hits.py]: I may have missed where the manhattan option comes into play here? Here is the function used by `get_all_hits` (called on line 62 here):

https://github.com/statgen/pheweb/blob/1d7d25c7b008a55254195b56f79b3f089e206df3/pheweb/load/top_hits.py#L17

Mainly asking as a check on my own understanding- I'm still finding some of what the pipeline does. AFAIK the `manhattan_peak_pval_threshold` only gets used in manhattan.py.

**Pull request merged**